Formula/redis@1.3.rb

```diff
@@ -0,0 +1,65 @@
+class Redis1310 < Formula
+  desc "Persistent key-value database, with built-in net interface"
+  homepage "http://redis.io/"
+  url "https://github.com/antirez/redis/archive/v1.3.12.tar.gz"
+  sha256 "326d8866317f7bcaafba80409b847700620025dbd34e638f85f05bc1c13beabd"
+  head "https://github.com/antirez/redis.git", :branch => "unstable"
```

Comment:
I couldn't find the branch for `1.3` and so I have kept the default branch for the repo. Please advice!